frontend/src/js/controllers/permissionCtrl.js

```diff
@@ -7,13 +7,34 @@
         .module('evalai')
         .controller('PermCtrl', PermCtrl);
 
-    PermCtrl.$inject = ['utilities'];
+    PermCtrl.$inject = ['utilities', '$rootScope'];
 
-    function PermCtrl(utilities) {
+    function PermCtrl(utilities, $rootScope) {
         var vm = this;
 
         // message for not verified users
         vm.emailError = utilities.getData('emailError');
-    }
 
+         vm.sendMail = false;
+        // Function to request a new verification email.
+        vm.requestLink = function () {
+            var userKey = utilities.getData('userKey');
+            var parameters = {};
+            parameters.url = 'accounts/user/resend-email';
+            parameters.method = 'POST';
+            parameters.token = userKey;
+            parameters.callback = {
+                onSuccess: function(response) {
+                    vm.sendMail = true;
+                    $rootScope.notify("success", "The verification link was sent again.");
+                },
+                onError: function(response) {
+                    var details = response.data
+                    if(response.status==429){$rootScope.notify("error", details.message);}
```

Comment:
Please fix the indentation and change the if block to `if (response.status == 429)`